pkg/client/ocm/client.go

```diff
@@ -103,7 +106,10 @@ func NewOCMConnection(ocmConfig *OCMConfig, BaseUrl string) (*sdkClient.Connecti
 }
 
 func NewClient(connection *sdkClient.Connection) Client {
-	return &client{connection: connection}
+	return &client{
+		connection: connection,
+		cache:      cache.New(15*time.Minute, 5*time.Minute),
```

Comment:
Let me know if the expiration and cleanup interval needs to be increased/decreased.

Reply:
We can increase the expiration time, to a day or an even bigger value :-D

Reply:
Since it doesn't really change we could even go for a week? :thinking: wdyt? or is that too long :joy:

Reply:
+1. No concern of going for a week.